app/controllers/users/phone_setup_controller.rb

```diff
@@ -70,13 +78,29 @@ def handle_create_success(phone)
           phone: @new_phone_form.phone,
           selected_delivery_method: @new_phone_form.otp_delivery_preference,
           phone_type: @new_phone_form.phone_info&.type,
+          selected_default_number: @new_phone_form.otp_make_default_number,
```

Comment:
I noticed a slight difference from `main`, the `selected_default_number` parameter is now always present in the URL when confirming a number, and it's `false` when confirming the first phone number. I don't think this makes a meaningful impact, since I checked that in `main`, the absence of the parameter is effectively the same as it being `false`.